submissions/TaniaBondarenko/html-movie-seat-booking/styles.css

```diff
@@ -0,0 +1,362 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background-color: #9b9b9b;
+}
+
+.mainContainer {
+  display: flex;
+  width: 70%;
+  height: auto;
+  flex-direction: row;
+  justify-content: center;
+  margin: auto;
+  background-color: #f6edea;
+}
+
+.screenImg {
+  fill: grey;
+}
+
+.screenImg:hover {
+  fill: black;
+}
+
+.screenText {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
Set `font-family` to the root element(body) and then just inherit them for tags who need it.

Reply:
Okay